### DISCUSSION TITLE
user experience improvements

### DISCUSSION THREAD

**Discussion by rgknox** [NGEET/fates#1012] [Ideas]
Discussions at the CLM land-model group, Jan 27 2022, generating ideas about improving user experience. Feel free to add ideas on this thread:

Lets create a Quick-start Guide

Lets create a Searchable User's Guide

**Comment by KatieMurenbeeld** [comment]
Something that links the equations/text to the parameters in the .F90 files.

**Comment by glemieux** [comment]
Include a "Noteworthy Differences" page comparing the differences between big-leaf clm and fates as part of the FAQ, akin to this: https://docs.julialang.org/en/v1/manual/noteworthy-differences/.

**Comment by glemieux** [comment]
> Something that links the equations/text to the parameters in the .F90 files.

I wonder if there is a package out there that will parse for docstrings in fortran :thinking:.  Does sphinx or doxygen have their own package extensions?

EDIT: https://sphinx-fortran.readthedocs.io/en/latest/user.autodoc.html looks promising

**Comment by glemieux** [comment]
Ok, here's a first crack at converting the wiki pages into a standard readthedocs (rtd) format: https://fates-users-guide.readthedocs.io/en/latest/index.html#

This took me about 6 hours total to pull together.  It only includes the 2019 tutorial and the faq in the user's guide section and the style guide in the developer's guide section so far.  A bunch of that was fussing around with rtd, but reformatting for the restructuredtext to a good chunk of time too.  Please take a look and let me know if this looks promising before I spend more time on it.

Note that its currently hosted on my personal [github account](https://github.com/glemieux/fates-users-guide) and my personal rtd account, hence the side bar ads.  If people think this looks worthwhile, I'll migrate it over to the NGEET org github and rtd accounts.
